requirements_test.txt

```diff
@@ -1,3 +1,4 @@
+autopep8
```

Comment:
I used pylint and black to catch issues and autopep8 to fix them in the past but we are not enforcing this because different developers have different aesthetic preferences - but maybe we should?